packages/graphql-toolbox/src/modules/SchemaView/ActionElementsBar.tsx

```diff
@@ -30,23 +30,20 @@ interface Props {
 }
 
 export const ActionElementsBar = ({ hasSchema, loading, onSubmit }: Props) => {
+    const tooltipRef = useRef<HTMLDivElement | null>(null);
     return (
         <div className="flex items-center h-12 w-full px-6">
-            <div className="justify-start">
-                <ProTooltip
-                    tooltipText="Build the schema to use the editor"
-                    arrowPositionOverride="left"
-                    blockVisibility={hasSchema}
-                    width={220}
-                    left={253}
-                    top={-3}
-                >
-                    <ViewSelectorComponent
-                        key="schema-editor-view-selector"
-                        elementKey="schema-editor-view-selector"
-                        isEditorDisabled={!hasSchema}
-                    />
-                </ProTooltip>
+            <div className="justify-start" ref={tooltipRef}>
+                <ViewSelectorComponent
+                    key="schema-editor-view-selector"
+                    elementKey="schema-editor-view-selector"
+                    isEditorDisabled={!hasSchema}
+                />
+                {!hasSchema ? (
+                    <SmartTooltip allowedPlacements={["right"]} style={{ width: "14rem" }} ref={tooltipRef}>
```

Comment:
```suggestion
                    <SmartTooltip allowedPlacements={["right"]} className="w-56" ref={tooltipRef}>
```
I haven't checked it but this tailwind CSS would be equivalent and keep in line with the rest of the code.